api/Headers.json

```diff
@@ -113,9 +113,10 @@
           "deprecated": false
         }
       },
-      "append": {
+      "Headers": {
```

Comment:
Ah, looks like the git diff butchered this one up -- it appears that you have potentially changed the feature identifiers, thus requiring an alteration to the sorting, correct?  We should probably separate this into its own PR for better review!